eth1.tf

```
@@ -15,3 +15,25 @@ module "nimbus_eth1_node_innova" {
 
   ips = ["194.33.40.237"] # metal-01.ih-eu-mda1.nimbus.eth1
 }
+
+/* Hetzner
+ * Dedicated AX42 Server
+ * Location: Finland, HEL1
+ * 1 x Primary IPv4
+ * 1 x 2 TB NVMe SSD
+ * 8 Core CPU
+ * 64 GB DDR5 ECC RAM
+*/
+
+module "nimbus_eth1_node_hetzner" {
+  source = "github.com/status-im/infra-tf-dummy-module"
+
+  name   = "metal"
```

We should call them something related to the purpose, for example:
```suggestion
  name   = "bench"
```